MANIFEST.in

```diff
@@ -3,9 +3,12 @@ include CONTRIBUTING.rst
 include HISTORY.rst
 include LICENSE
 include README.rst
+include Makefile
+
+exclude .editorconfig requirements_dev.txt
 
 recursive-include tests *
```

Comment:
See `graft` comment in https://github.com/azavea/django-amazon-ses/commit/f4903b2a711287884b3244a5e4f461c723c2ad5f.